packages/bpk-component-image/src/BpkImage.js

```diff
@@ -134,6 +137,14 @@ class BpkImage extends Component<BpkImageProps> {
       classNames.push(getClassName('bpk-image--no-background'));
     }
 
+    if (borderRadiusStyle) {
```

Comment:
Now that `borderRadiusStyle` defaults to `none`, this `if` statement will always be matched. To preserve the logic you'd need to change it:

```suggestion
    if (borderRadiusStyle !== BORDER_RADIUS_STYLES.none) {
```